internify/src/components/molecules/styles/LoginForm.css

```diff
@@ -3,10 +3,20 @@
   flex-direction: column;
   align-items: center;
   padding: 3em 0;
+  background: linear-gradient(
+      66.05deg,
+      #e61a4d -19.05%,
+      rgba(255, 255, 255, 0) 42.54%
+    ),
+    linear-gradient(199.29deg, #e61a4d -2.31%, rgba(255, 255, 255, 0) 77.09%),
+    #043a74;
+  
 }
 
+
 .login_form_container {
   width: 330px;
+  height: 500px;
```

Comment:
in the future, refrain from creating fixed heights for containers unless you know for sure that the height is going to be fixed, period. Usually the height should be flexible with the amount of components that are in it